packages/app-scripts-preact/snowpack.config.js

```diff
@@ -21,4 +21,5 @@ module.exports = {
   installOptions: {
     installTypes: isTS,
   },
+  plugins: ["@prefresh/snowpack"]
```

Comment:
Adding "@snowpack/plugin-babel" to this list will connect Babel to your build.

You'll also want to create a babel.config.json in your template to extend from the babel config you see defined there: 

```js
// templates/app-template-preact/babel.config.js
{"extends": "@snowpack/app-scripts-preact/babel.config.json"}
```

Let me know if you run into any trouble here and I can help debug

Reply:
Would you advise also adding `preset-env` and stuff if this is used for production? Or are we not transpiling down?

Reply:
I'm writing the docs to explain this as we speak :) But no, optimizations like preset-env, minification, etc. are all automatically handled for you by Snowpack.

You can see this in action via `snowpack build --bundle`. In the future, we'll offer automatic production optimizations on `snowpack build --no-bundle` as well.

Reply:
That sounds wonderful, I've been enjoying building this demo out as well so must say Snowpack rocks! Will finish the integration tongiht with this new information. Thanks!

Reply:
When using `babel` the contents I receive in my plugin seem to be `undefined`

Reply:
hmm, mind pushing up your latest so that I can take a look?

Reply:
Also, make sure that you rebase on top of master & re-run `yarn install` in the repo root, to get the latest version of Snowpack installed in case somethings outdated

Reply:
Pushed, https://github.com/pikapkg/create-snowpack-app/pull/54/files#diff-fe0b3c8a3d892219f073e1d1579c0144R10 is undefined

Reply:
Nice, it looks like you found & fixed the issue. If you rebase onto master & re-run `yarn install` it looks like it should all work now.